.github/workflows/sync-files.yml

```diff
@@ -0,0 +1,20 @@
+name: Check synchronized files
```

Comment:
Doesn't a workflow like this already exist?

Reply:
Not in the codeql repository. There is likely an indirect check in the closed source CLI repository.

Reply:
This probably isn't the right branch to be adding new workflows, but it seems both harmless and useful, so let's keep it.